doc/HttpLuaModule.wiki

```diff
@@ -5447,7 +5454,7 @@ Fetch a list of the keys from the dictionary, up to <code><max_count></code>.
 
 By default, only the first 1024 keys (if any) are returned. When the <code><max_count></code> argument is given the value <code>0</code>, then all the keys will be returned even there is more than 1024 keys in the dictionary.
 
-'''WARNING''' Be careful when calling this method on dictionaries with a really huge number of keys. This method may lock the dictionary for quite a while and block all the nginx worker processes that are trying to access the dictionary.
+'''CAUTION''' Avoid calling this method on dictionaries with a very large number of keys as this method may lock the dictionary for a significant amount of time while blocking nginx worker processes that are trying to access the dictionary.
```

Comment:
very long sentence and yet important. Could use some punctuation maybe?
```
for a significant amount of time[, all the] while blocking Nginx worker processes
```

note: capitalized "Nginx" (if desired).